app/models/tasks/cavc_poa_clarification_task.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Task for the CAVC Litigation Support team to clarify the Power of Attorney on a remand before sending the 90 day
+# letter to the veteran
+class CavcPoaClarificationTask < Task
```

Comment:
Add validation for data integrity
```suggestion
class CavcPoaClarificationTask < Task
  validates :parent, presence: true, parentTask: { task_type: SendCavcRemandProcessedLetterTask }, on: :create
```